ote/src/clj/ote/services/admin.clj

```diff
@@ -20,15 +24,33 @@
   (nap-users/list-users db {:email (str "%" query "%")
                             :name (str "%" query "%")}))
 
-(defn- admin-routes [db http]
+(defn- admin-delete-transport-service!
+  "Allow admin delete single transport service by id"
+  [nap-config db user id]
+  (let [return (transport/delete-transport-service! nap-config db user id)
+        auditlog {::auditlog/event-type :delete-service
+                  ::auditlog/event-attributes
+                                      [{::auditlog/name "transport-service-id", ::auditlog/value (str id)}]
```

Comment:
Add transport-service-name parameter that is the name of the service. That way we can later show a better description of what was deleted.